src/app/admin/emulators/[emulatorId]/custom-fields/EmulatorCustomFieldsClientPage.tsx

```
@@ -0,0 +1,215 @@
+'use client'
+
+import { PlusCircle, Copy, FileJson, ShieldAlert } from 'lucide-react'
+import Link from 'next/link'
+import { useState } from 'react'
+import UnauthenticatedPage from '@/components/auth/UnauthenticatedPage'
+import { Button, LoadingSpinner } from '@/components/ui'
+import { api } from '@/lib/api'
+import { hasPermission, PERMISSIONS } from '@/utils/permission-system'
+import { hasRolePermission } from '@/utils/permissions'
+import { Role } from '@orm'
```

ESLint

[error] `@orm` import should occur before import of `lucide-react`

(import/order)